euth/comments/templatetags/react_comments.py

```diff
@@ -13,6 +15,10 @@
 def react_comments(context, obj):
     request = context['request']
 
+    serializer = ThreadSerializer(
+        obj.comments.all(), many=True, context={'request': request})
```

Comment:
@slomo can we assume that everybody who wants to use the comments adds a comments generic relation to their model?